src/renderer/redux/selectors/publish.js

```diff
@@ -25,3 +25,49 @@ export const selectPendingPublish = uri =>
       publish => (publish.name === claimName || publish.name === contentName) && !publish.isEdit
     )[0];
   });
+
+// Is the current uri the same as the uri they clicked "edit" on
+export const selectIsStillEditing = createSelector(selectPublishFormValues, publishState => {
+  const { editingURI, uri } = publishState;
+
+  const { isChannel: currentIsChannel, claimName: currentClaimName, contentName: currentContentName } = parseURI(uri);
+  const { isChannel: editIsChannel, claimName: editClaimName, contentName: editContentName } = parseURI(editingURI);
+  
+  // Depending on the previous/current use of a channel, we need to compare different things
+  // ex: going from a channel to anonymous, the new uri won't return contentName, so we need to use claimName
+  if (!currentIsChannel && editIsChannel) {
+    return currentClaimName === editContentName;
+  } else if (currentIsChannel && !editIsChannel) {
+    return currentContentName === editClaimName;
+  } else if (!currentIsChannel && !editIsChannel) {
+    return currentClaimName === editClaimName;
+  } else {
+    return currentContentName === editContentName;
+  }
```

Comment:
I don't like the above code, is there a better way to do this?

Reply:
Ensures that users are still editing if they have the same claim name, but different channels (anonymous => channel)